src/service/logger/logger.development.ts

```diff
@@ -0,0 +1,119 @@
+[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;99mf[39m[38;5;99mf[39m[38;5;99mL[39m[38;5;99mL[39m[38;5;99mL[39m[38;5;99mL[39m[38;5;99m1[39m[38;5;99m1[39m[38;5;60m [39m[38;5;60m [39m[38;5;16m [39m[38;5;16m [39m[38;5;60m,[39m[38;5;60m,[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60m,[39m[38;5;60m,[39m[38;5;16m [39m[38;5;16m [39m[38;5;60m:[39m[38;5;60m:[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60m:[39m[38;5;60m:[39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;60m:[39m[38;5;60m:[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;16m [39m[38;5;16m [39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m,[39m[38;5;60m,[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m,[39m[38;5;60m,[39m[38;5;60mi[39m[38;5;60mi[39m
+[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;99mf[39m[38;5;99mf[39m[38;5;99mL[39m[38;5;99mL[39m[38;5;99mL[39m[38;5;99mL[39m[38;5;99m1[39m[38;5;99m1[39m[38;5;60m [39m[38;5;60m [39m[38;5;16m [39m[38;5;16m [39m[38;5;60m:[39m[38;5;60m:[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m:[39m[38;5;60m:[39m[38;5;16m [39m[38;5;16m [39m[38;5;60m:[39m[38;5;60m:[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;16m [39m[38;5;60m:[39m[38;5;60m:[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60mi[39m[38;5;60m:[39m[38;5;60m:[39m[38;5;16m [39m[38;5;16m [39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;16m [39m[38;5;16m [39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m;[39m[38;5;60m [39m[38;5;60m [39m[38;5;60m,[39m[38;5;60mi[39m[38;5;60mi[39m
+            
+--------------------------------------------------------------------------------`
+
+const WELCOME_MESSAGES = [
+  `
+              Hi! 👋 Welcome to Tatum, the Javascript SDK for Web3.
```

Comment:
Same as above - I would probably either take some common parts into variable or make a function out of it instead to reduce some code duplication, but I understand that it is not that important to do so in this case.